assets/src/helpers/bsIcons.tsx

```diff
@@ -145,19 +145,22 @@ export const ExclamationCircleFill = (props: SvgProps) => (
 /**
  * @returns https://icons.getbootstrap.com/icons/exclamation-triangle-fill/
  */
-export const ExclamationTriangleFill = (props: SvgProps) => (
-  <svg
-    xmlns="http://www.w3.org/2000/svg"
-    width="16"
-    height="16"
-    fill="currentColor"
-    className="bi bi-exclamation-triangle-fill"
-    viewBox="0 0 16 16"
-    aria-hidden
-    {...props}
-  >
-    <path d="M8.982 1.566a1.13 1.13 0 0 0-1.96 0L.165 13.233c-.457.778.091 1.767.98 1.767h13.713c.889 0 1.438-.99.98-1.767zM8 5c.535 0 .954.462.9.995l-.35 3.507a.552.552 0 0 1-1.1 0L7.1 5.995A.905.905 0 0 1 8 5m.002 6a1 1 0 1 1 0 2 1 1 0 0 1 0-2" />
-  </svg>
+export const ExclamationTriangleFill = forwardRef(
```

Comment:
Is it possible to do this by attaching the `forwardRef` to a parent element of the icon, rather than the bsIcon itself?

Reply:
Oh yeah 💯 . I got stumped on this earlier and am still stumped. I just pushed up a commit with a few variations of the icon with `forwardRef` combos. I changed the `SvgProps` to allow refs to be passed, and set up an AlertThing component that forwards the ref, but even with that plumping, if I use either `ExclamationTriangleFillJustSvg` or the wrapper `ExclamationTriangleFillWithRef` in `<AlertThing />`, neither work as I would hope?? only `ExclamationTriangleFill`. I dunno what to try next

Reply:
What if you instead wrap the `<SVG/>` in a `<div/>` under the `<Tippy/`?

Reply:
NUH UH. I thought if any nested component was under <Tippy/> then it would have problems 😭  and all this time, Tippy just didn't want an immediately nested component. Thank youuuuu 🥲

Reply:
I believe what is happening is that Tippy wants to register an `onClick` handler and isn't opinionated about what element is the container of the tooltip, so it uses `ref` to access the associated DOM element. Because the SVG's don't expose their ref's by default, Tippy wasn't able to register the click handler which I believe is an error?

So by giving it a direct reference to another element that does have a ref, you can avoid needing that on the SVGIcons.

This does bring up the question of if it's worth making refs work with our SVG's so we don't have to do this "wrapping" on the icon, just to make our icon components a little more complete?

Reply:
Something for a future ticket I suppose